core/src/main/java/org/opensearch/sql/storage/StorageEngineFactory.java

```diff
@@ -0,0 +1,21 @@
+/*
+ *
+ *  * Copyright OpenSearch Contributors
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.opensearch.sql.storage;
+
+import java.net.URISyntaxException;
+import java.util.Map;
+import org.opensearch.sql.catalog.model.ConnectorType;
+
+public interface StorageEngineFactory {
+
+  ConnectorType getConnectorType();
```

Comment:
I think we may need to rename either one for consistency sooner or later. Although only *Connector* concept is exposed to customer, it may be confusing for us developer.

Reply:
rename to getConnector?

Reply:
Will convert StorageEngine terminology to connector in future.

Reply:
https://github.com/opensearch-project/sql/issues/1002